coders/webp.c

```diff
@@ -232,11 +232,21 @@ static int FillBasicWEBPInfo(Image *image,const uint8_t *stream,size_t length,
 }
 
 static int ReadSingleWEBPImage(Image *image,const uint8_t *stream,
-  size_t length,WebPDecoderConfig *configure,ExceptionInfo *exception)
+  size_t length,WebPDecoderConfig *configure,ExceptionInfo *exception,MagickBooleanType is_first)
 {
   int
     webp_status;
 
+  size_t
```

Comment:
Thanks for ordering the properties but we also order the type. Could you move this below `register unsigned char` and there is also already a `ssize_t` block.